stripe/src/main/java/com/stripe/android/StripeApiHandler.java

```diff
@@ -946,6 +694,107 @@ private static void handleAPIError(String rBody, int rCode, String requestId)
         }
     }
 
+    private static String requestData(
+            @RestMethod String method,
+            String url,
+            Map<String, Object> params,
+            RequestOptions options)
+            throws AuthenticationException, InvalidRequestException,
+            APIConnectionException, CardException, APIException {
+
+        if (options == null) {
+            return null;
+        }
+
+        String originalDNSCacheTTL = null;
+        Boolean allowedToSetTTL = true;
+
+        try {
+            originalDNSCacheTTL = java.security.Security
+                    .getProperty(DNS_CACHE_TTL_PROPERTY_NAME);
+            // disable DNS cache
```

Comment:
This only applies to the connection in an app using this library. This was also part of the legacy style code -- if we were able to use any libraries, I'd just let OkHttp handle all this stuff.